help/moderation/appeals.md

```diff
@@ -0,0 +1,49 @@
+# Appealing a moderation decision 
+
+It is possible to appeal the classification of an announced article or the decision to decline a submission. In some cases, we may require that a paper be accepted for publication in a conventional journal before an appeal is considered. Note that some works that have been published or accepted by a journal may still be declined at arXiv’s discretion.
+
+If you disagree with a [moderation](/help/moderation) decision, you may appeal for your desired outcome. The appeals process allows moderators time to reconsider a submission, but repeated appeals with no additional information cannot be considered.
+
+Appeals should be sent through our [user support portal](https://arxiv.org/support/moderation_help). If you cannot access the portal you can contact The arXiv Content Management & User Support Team by email at moderation@arxiv.org. 
+
+- [What is a moderation appeal?](#what-is)
+- [What happens when I appeal?](#what-happens)
+- [What happens if my appeal is accepted?](#accepted)
+- [What if my appeal is denied?](#denied)
+- [Can all decisions be appealed?](#final)
+
+<span id="what-is"></span>
+## What is a moderation appeal?
+
+An appeal is a message sent to The arXiv Content Management & User Support Team. You are not communicating directly with the original moderators. In general, appeals are for additional scrutiny of moderation [decisions before](help/moderation#what-policies) and [after announcement](/help/moderation#policies-after). No third-party appeal for inclusion will be considered. 
```

Comment:
```suggestion
The appeal request will be sent to the arXiv User Support Team. You are not communicating directly with the original moderators. In general, appeals are for additional scrutiny of moderation [decisions before](help/moderation#what-policies) and [after announcement](/help/moderation#policies-after). No third-party appeal for inclusion will be considered. 
```

Reply:
I wonder if we should add here and elsewhere that contacting moderators directly is not acceptable.

Reply:
We could. I thought it was pretty obvious from the tone of the page how it worked, and we say that in a few spots. Someone getting this far in our documentation would have seen that before, but we could add it for when we're just dumping it into our rejection messages.